extensions/amp-analytics/0.1/vendors/colanalytics.json

```diff
@@ -2,12 +2,23 @@
   "requests": {
     "host": "https://ase.clmbtech.com",
     "base": "${host}/message",
-    "pageview": "${base}?cid=${id}&val_101=${id}&val_101=${canonicalPath}&ch=${canonicalHost}&uuid=${uid}&au=${authors}&zo=${zone}&sn=${sponsorName}&ct=${contentType}&st=${scrollTop}&sh=${scrollHeight}&dct=${decayTime}&tet=${totalEngagedTime}&dr=${documentReferrer}&plt=${pageLoadTime}&val_108=${title}&val_120=3"
+    "pageview": "${base}?cid=${id}&val_101=${id}&val_101=${int}${canonicalPath}&ch=${canonicalHost}&val_108=${title}&val_120=3&val_102=${sourceUrl}&val_124=${externalReferrer}&uuid=${uid}",
+    "event": "${base}?cid=${id}&val_101=${id}&val_101=${int}${canonicalPath}&val_101=${tet}${totalEngagedTime}&val_102=${clk}${sourceUrl}"
```

Comment:
remove `${totalEngagedTime}`?

Reply:
${totalEngagedTime} is not removed only moved to click level event since in pageview event value will be '0' (as per the variable definition).